resources/sass/abstracts/_mixins.scss

```diff
@@ -0,0 +1,90 @@
+@mixin modal-content {
+    margin-left: 18% !important;
+    //margin-right: 20% !important;
+    width: 63% !important;
+}
+
+@mixin form-left-text {
+    font-size: 16px;
```

Comment:
I dont see a consistent style guide implemented across the website. 

Things like, margins, paddings, fonts and other consistent css elements should have only one source of truth